lib/teamsnap.rb

```diff
@@ -104,6 +104,10 @@ def init(opts = {})
         )
       end
 
+      unless opts.include?(:backup_cache) && opts[:backup_cache] == false
+        opts[:backup_cache_file] = TeamSnap.backup_file(opts[:backup_cache])
+      end
+
```

Comment:
Actually, you could probably go simpler and just do

`opts[:backup_cache_file] = opts.fetch(:backup_cache) { "./tmp/.teamsnap_rb" }`